datasets/nlcd.data.mdx

```diff
@@ -0,0 +1,286 @@
+---
+id: nlcd-annual-conus
+name: 'National Land Cover Database LULC Classifications'
+description: "National Land Cover Database Land Use - Land Cover classifications for CONUS, 2001-2021 at 30 m resolution."
+
+media:
+  src: ::file ./nlcd-cover.png
+  alt: Boston, MA skyline.
+  author: 
+    name: Eric Kilby
+    url: https://openverse.org/image/a9e174e6-d4e1-4377-81e2-6e71bf1a9602?q=skyline
+taxonomy:
+  - name: Topics
+    values:
+      - Agriculture
+      - Biomass
+      - Environmental Justice
+      - Land Cover
+  - name: Source
+    values:
+      - MRLC
+layers:
+  - id: nlcd-annual-conus
+    stacApiEndpoint: https://openveda.cloud/api/stac
+    tileApiEndpoint: https://openveda.cloud/api/raster
```

Comment:
```suggestion
```
temporary staging url workaround no longer needed, we can use the default configured stac api and tiler

Reply:
are you suggesting to remove these two lines entirely?

Reply:
yeah, I added them in this [PR](https://github.com/NASA-IMPACT/veda-config/pull/450/files) when we didn't have the nlcd custom colormap in production but now we do. Interestingly, the [nlcd-annual-conus preview ](https://deploy-preview-450--visex.netlify.app/exploration?search=&datasets=%5B%7B%22id%22%3A%22nlcd-annual-conus%22%2C%22settings%22%3A%7B%22isVisible%22%3Atrue%2C%22opacity%22%3A100%2C%22analysisMetrics%22%3A%5B%7B%22id%22%3A%22mean%22%2C%22label%22%3A%22Average%22%2C%22chartLabel%22%3A%22Average%22%2C%22themeColor%22%3A%22infographicB%22%7D%2C%7B%22id%22%3A%22std%22%2C%22label%22%3A%22St+Deviation%22%2C%22chartLabel%22%3A%22St+Deviation%22%2C%22themeColor%22%3A%22infographicD%22%7D%5D%7D%7D%5D&taxonomy=%7B%7D&date=2021-01-01T07%3A00%3A00.000Z)for that PR is working.